### PR TITLE
Implementation for sickragetv/sickrage-issues#322 - Add extra progress-bar color for when show is fully downloaded

### PR DESCRIPTION
**Pull request by sn3akz**
Added a different shade of green for when and only when the show is 100% downloaded

### CODE REVIEW

File: gui/slick/interfaces/default/home.tmpl [hunk 1]
```diff
@@ -364,7 +364,10 @@ $myShowList.sort(lambda x, y: cmp(x.name, y.name))
 					}
 					if (classvalue>80) {
```
Comment (fernandog) [inline gui/slick/interfaces/default/home.tmpl]: @sn3akz 
Here should be:
if (classvalue>80 && classvalue<100) {

Comment (sn3akz) [inline gui/slick/interfaces/default/home.tmpl]: Could be, but doesn't do much. In my code, the classtoadd variable just gets overridden when it's a 100. Nothing wrong with that.

The old code should indeed be >=. When it's 80, no color is selected. Maybe that's also the cause of issue sickrage-issues/#303

**Comment by fernandog** [comment]
@sn3akz can you update the code and test the issue 303?

**Comment by sn3akz** [comment]
Will do

**Pull request merged**